src/components/DateInput.jsx

```diff
@@ -136,14 +157,12 @@ export default class DateInput extends React.Component {
           disabled={disabled}
           readOnly={isTouch}
           required={required}
-          aria-describedby={screenReaderMessage && screenReaderMessageId}
+          aria-describedby={screenReaderMessageId}
         />
 
-        {screenReaderMessage &&
-          <p id={screenReaderMessageId} className="screen-reader-only">
-            {screenReaderMessage}
-          </p>
-        }
+        <p id={screenReaderMessageId} className="screen-reader-only">
```

Comment:
since `screenReaderMessage` isn't required, it seems like you should only render this if it exists.